frontend/src/components/shared/StreamlitMarkdown.test.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license
+ * Copyright 2018-2019 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { ReactElement } from "react"
+import ReactMarkdown from "react-markdown"
+
+import {
+  linkWithTargetBlank,
+  linkReferenceHasParens,
+} from "./StreamlitMarkdown"
+import { create } from "react-test-renderer"
+
+// Fixture Generator
+const getMarkdownElement = (body): ReactElement => {
+  const renderers = {
+    link: linkWithTargetBlank,
+    linkReference: linkReferenceHasParens,
+  }
+  return <ReactMarkdown source={body} renderers={renderers} />
+}
+
+describe("linkReference", () => {
+  it("renders a link with _blank target", () => {
+    const body =
+      'Some random URL like [Streamlit](https://streamlit.io" target="_blank")'
+    const component = create(getMarkdownElement(body))
```

Comment:
as we are currently using enzyme , you could do this using `mount()`

Reply:
Can you give me an example?

Reply:
This would be an example:
```
it("renders a link with _blank target", () => {
    const body =
      'Some random URL like [Streamlit](https://streamlit.io" target="_blank")'
    const component = mount(getMarkdownElement(body))
    expect(component.find("a[target='_blank']").props().href).toBe(
      "https://streamlit.io"
    )
  })
```